src/schemes/fluid/weakly_compressible_sph/rhs.jl

```diff
@@ -153,3 +165,70 @@ end
 
     return p_a, p_a
 end
+
+@inline function calc_surface_tension(particle, neighbor, pos_diff, distance,
+                                      particle_system::FluidSystem,
+                                      neighbor_system::FluidSystem,
+                                      surface_tension_a::CohesionForceAkinci,
+                                      surface_tension_b::CohesionForceAkinci)
```

Comment:
```suggestion
@inline function surface_tension_force(surface_tension_a::CohesionForceAkinci,
                                      surface_tension_b::CohesionForceAkinci,
                                      particle_system::FluidSystem,
                                      neighbor_system::FluidSystem,
                                      particle, neighbor, pos_diff, distance)
```
1. I think we only used `calc_` for functions that compute something and write it into some array, while this here returns a force.
2. I think this parameter order would be most consistent with the rest of TrixiP.
3. Do you think it would be easier for new developers to find these functions when they're inside `surface_tension.jl` instead of here?
4. Same for `calc_adhesion` below.